Branch-SDK/src/main/java/io/branch/referral/ServerRequestQueue.java

```diff
@@ -148,26 +143,8 @@ void enqueue(ServerRequest request) {
     }
     
     /**
-     * <p>Removes the queued {@link ServerRequest} object at position with index 0 within the queue,
-     * and returns it as a result.</p>
-     *
-     * @return The {@link ServerRequest} object at position with index 0 within the queue.
-     */
-    ServerRequest dequeue() {
```

Comment:
This is part of the change made to BranchPostTask, onPostExecute, I now use queue.remove(request).